src/MainTest.java

```diff
@@ -18,4 +16,58 @@ public void testCalculateArea() {
         double actualArea = rectangle.calculateArea();
         Assertions.assertEquals(expectedArea, actualArea);
     }
+
+
+    @Test
```

Comment:
do we need to test java getter and setter?

Reply:
According to TDD, before developing any new features, we should first write unit tests.

Reply:
i know but even for getter and setter? i approved but please search this issue if its needed or not